### PR TITLE
Create moving.md

### PR DESCRIPTION
**Pull request by wideaperture**
A brief guide for moving to a new computer or OS install. I wasn't quite sure where to place this. It's too big to fit neatly alongside the parsimonious answers in the FAQ document. It doesn't seem to be a subtopic of any of the existing docs. The other documents most like it are in the "Getting Started" section, but this is obviously aimed at folks who've been using Zettlr for a while (and perhaps those who are hesitant to commit until they have a sense of how portable their data will be). "References" seemed like the best fit, but it still feels different than the other documents there. If this doc gets accepted feel free to place it wherever it makes the most sense.

### CODE REVIEW

**Comment by nathanlesage** [comment]
First of all, thank you for doing this work! The guide already reads pretty good, but I do have a few remarks:

* First of all, I think your choice of the reference section is absolutely right
* Then, you'll also need to adapt the configuration file for the english documentation (I think it's at `/config/en.yaml`) and add it to the navigation so that people can find it
* Also, I think it makes more sense to tell the users to first move the files because that's probably either way something they'll do first, so maybe switch those two sections
* Lastly, I don't know but something feels off about the "moving" description. Maybe … migrating? I don't know, but the word feels a tad off.

File: docs/en/reference/migrating.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+Because Zettlr workspaces are simply folders on your computer and your notes are simply text files, moving this data to a new computer is as simple as copying these folders from one computer to the next. All of the metadata about your notes and the connections between them — note IDs, tags, etc. — is saved to the individual note files themselves, meaning this information travels right along with the files.
+
+Once you've copied the workspace folders containing your notes from your old computer and placed them on the new machine, you can add each workspace and its files to your new Zettlr install simply by going to the `File > Open Workspace …` menu option and selecting that workspace folder. Note that this is the same process you would use you if were adding any other folder to Zettlr. The workspace folder, including the notes it contains, will immediately be populated into your new Zettlr install. The connections between notes will also be retained, though if you have links to notes that live in different workspaces, you'll have to make sure you've likewise opened those workspace folders and their contents within your new Zettler install.
```
Comment (nathanlesage) [inline docs/en/reference/migrating.md:15]: I would argue that we shouldn't state this here because one would need to do that after copying the config file. Right now, some people may be inclined to do a fresh install, open their folders, then copy the config and be confused why their folders are gone again.

Maybe it might make sense to just end the "move files" section and continue with the configuration here.

File: docs/en/reference/migrating.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Any files or notes that aren't part of a workspace folder can also be opened in your new Zettlr install once you've moved them to your new machine, just as you would use Zettlr to open any other file.
```
Comment (nathanlesage) [inline docs/en/reference/migrating.md:17]: That info seems a bit redundant, and may confuse users, or do I overlook something here?

File: docs/en/reference/migrating.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+##  Copying Your Application Settings to the New Machine
+
+If you've customized Zettlr at all and/or you want to retain the various stats about your notes, there are a few configuration files that you may want to back up and copy over to your new machine. This data is located in the _application data path_ whose location depends on your Operating System:
```
Comment (nathanlesage) [inline docs/en/reference/migrating.md:21]: Maybe reword to something like "Since you've already been working with Zettlr, you'll want to also keep your settings and any statistics that are not contained in your files..." -- because I think most people don't touch Custom CSS

File: docs/en/reference/migrating.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+* `stats.json` (your writing statistics)
+* `custom.css` (your custom CSS, if any)
+* `tags.json` (your colour-coded tags)
+* `targets.json` (your writing targets)
+* `user.dic` (your own custom dictionary)
+* `defaults` (default settings for document import and export)
+* `snippets` (any snippets you've set up for auto-filling note content)
+
+Once you've installed Zettlr on your new computer and launched it for the first time, the software will create a new application path folder (see above). When you're ready to import the application settings to your new computer, close Zettlr and copy the backups of the files and subfolders you just made into the newly created application path folder. If you're prompted as to whether it's okay to replace existing files, you can say "okay." You've now copied over the application settings from your old computer.
```
Comment (nathanlesage) [inline docs/en/reference/migrating.md:37]: I would move that one up, below the section with moving files, so that a user sees "Ah, first I need to copy files. Then I need to install Zettlr, launch it, and close it again, and then copy those files".

Therefore I would also reword this to something more explicit with regard to what users have to do: "Install Zettlr and launch it. This will create the configuration directory into which you can then put your files. Then, quit the app again, and copy all the files you wish to keep from your old install into the new configuration directory."

Maybe additionally, we want to add some note of caution: "Zettlr will automatically remove the paths to the workspaces of your old setup, since they are not valid anymore. So when you start Zettlr after copying the files, you will be greeted with an empty window. Now you can open the workspaces again."

(Feel free to adapt the wording, I'm typing this on my phone)

**Review by nathanlesage — COMMENTED** [review]
We're almost there! I've left a few comments because I feel your write-up could use a little more "guide-style" so that people can literally follow the page while migrating! Feel free to look over the changes!

File: docs/en/reference/migrating.md (hunk 1 above)
Comment (wideaperture) [reply docs/en/reference/migrating.md:15]: I get what you're suggesting, but have a couple questions. Per our previous discussion, I removed the advice to copy `config.json` to the new machine. If the user refrains from copying this file over, is this still an issue? If not, would it be more straightforward to just have an aside about what happens if you _were_ to copy over `config.json`, so users who do so aren't confused, but to otherwise assume they won't be migrating this file?

File: docs/en/reference/migrating.md (hunk 2 above)
Comment (wideaperture) [reply docs/en/reference/migrating.md:17]: The guidance to this point was about moving workspace folders containing files. I just wanted to be sure to say something about any unfoldered files the user may be moving over. But if that's too obvious, this sentence could just be dropped or something similar folded into the existing text above.

File: docs/en/reference/migrating.md (hunk 3 above)
Comment (wideaperture) [reply docs/en/reference/migrating.md:21]:
```suggestion
Since you've already been working with Zettlr, you'll want to also keep your settings and any statistics that are not contained in your files. This data is located in the _application data path_ whose location depends on your Operating System:
```

File: docs/en/reference/migrating.md (hunk 1 above)
Comment (nathanlesage) [reply docs/en/reference/migrating.md:15]: I remember that I was like "Oh no don't copy the config" BUT then I realized that (a) there are the autocorrect keys stored in the config (not the best design I know) and (b) the config should automatically just remove all the links that are no longer valid.

So, bottom line is: Copying over the config should actually be fine.

File: docs/en/reference/migrating.md (hunk 2 above)
Comment (nathanlesage) [reply docs/en/reference/migrating.md:17]: Aaaaah you meant that. Well, I think unless one has the specific idea  in one's mind about the root files, it's probably best to just trust that when you tell people "Btw move all your files" that they do just that, and then it should be straightforward to open those "unfoldered" (funny term btw!) files again.

File: docs/en/reference/migrating.md (hunk 4 above)
Comment (nathanlesage) [reply docs/en/reference/migrating.md]: I've commented above – my problem was that when I wrote the first comment in my mind it was like "config should not be moved" and while I was writing this last comment my mind suddenly turned to "actually it should be fine" :D

File: docs/en/reference/migrating.md (hunk 1 above)
Comment (wideaperture) [reply docs/en/reference/migrating.md]: I added in guidance on copying `config.json` and reworked/reflowed the other guidance as necessary.

File: docs/en/reference/migrating.md (hunk 2 above)
Comment (wideaperture) [reply docs/en/reference/migrating.md]: I think this is resolved in the reworked version.

**Comment by nathanlesage** [comment]
Looks good now! Contains much information so that technically less versed users can feel safe doing it and it looks like there's everything in there! Thank you for the changes :)
